src/expr/elim_shadow_converter.cpp

```diff
@@ -91,7 +94,31 @@ Node ElimShadowNodeConverter::eliminateShadow(const Node& q)
   ElimShadowNodeConverter esnc(nm, q);
   // eliminate shadowing in all children
   std::vector<Node> children;
-  children.push_back(q[0]);
+  // drop duplicate variables
```

Comment:
If you are changing the quantifier prefix, please add that to the documentation.